charger/charger.go

```diff
@@ -1,6 +1,7 @@
 package charger
 
 import (
+	"errors"
```

Comment:
Bitte rebase

Reply:
Verstehe nicht

Reply:
Die Änderungen gehören nicht in diesen PR. Mittels `git rebase` zu lösen :)

Reply:
Ich habe kein lokales git - und github unterstützt wohl kein Rebase :-(

Reply:
Außerdem: #7723 ist doch geschlossen und wurde wieder reverted. Ich kann den doch gar nicht neu stellen?

Reply:
Sorry, aber das kriege ich nicht besser hin...